cmd/image-builder/main.go

```diff
@@ -57,6 +58,12 @@ func main() {
 
 	if conf.GlitchTipDSN == "" {
 		logrus.Warn("Sentry/Glitchtip was not initialized")
+	} else {
+		sentryhook := sentrylogrus.NewFromClient([]logrus.Level{logrus.PanicLevel,
+			logrus.FatalLevel, logrus.ErrorLevel},
+			sentry.CurrentHub().Client())
+		logrus.AddHook(sentryhook)
+		defer sentryhook.Flush(2 * time.Second)
```

Comment:
Why is the flush needed here?

Reply:
to process logrushook channels (going into sentry's channels), I believe.